### ISSUE TITLE
Crash on Dedicated Server

### ISSUE THREAD

**Issue by Tylian** [elytra/FruitPhone#1]
Server is throwing this error when ever I connect to it.

```
[07:33:38] [Server thread/ERROR] [FML/]: Exception caught during firing event net.minecraftforge.fml.common.gameevent.TickEvent$PlayerTickEvent@22fa3ca7:
java.lang.NoSuchMethodError: net.minecraft.entity.player.EntityPlayer.func_174824_e(F)Lnet/minecraft/util/math/Vec3d;
	at com.elytradev.fruitphone.FruitPhone.onPlayerTick(FruitPhone.java:316) ~[FruitPhone.class:?]
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_486_FruitPhone_onPlayerTick_PlayerTickEvent.invoke(.dynamic) ~[?:?]
	at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:90) ~[ASMEventHandler.class:?]
	at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:185) [EventBus.class:?]
	at net.minecraftforge.fml.common.FMLCommonHandler.onPlayerPreTick(FMLCommonHandler.java:360) [FMLCommonHandler.class:?]
	at net.minecraft.entity.player.EntityPlayer.func_70071_h_(EntityPlayer.java:189) [zs.class:?]
	at net.minecraft.entity.player.EntityPlayerMP.func_71127_g(EntityPlayerMP.java:303) [lu.class:?]
	at net.minecraft.network.NetHandlerPlayServer.func_73660_a(NetHandlerPlayServer.java:162) [me.class:?]
	at net.minecraftforge.fml.common.network.handshake.NetworkDispatcher$1.func_73660_a(NetworkDispatcher.java:213) [NetworkDispatcher$1.class:?]
	at net.minecraft.network.NetworkManager.func_74428_b(NetworkManager.java:287) [eo.class:?]
	at net.minecraft.network.NetworkSystem.func_151269_c(NetworkSystem.java:180) [md.class:?]
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:732) [MinecraftServer.class:?]
	at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:387) [ld.class:?]
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:613) [MinecraftServer.class:?]
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:471) [MinecraftServer.class:?]
	at java.lang.Thread.run(Unknown Source) [?:1.8.0_77]
```

Pretty sure EnitityPlayer.getPositionEyes is a client-sided method. :p

**Comment by exaskye** [comment]
Oh god damn it I forgot to test it on a dedicated server.

**Comment by exaskye** [comment]
It's not client-only in 1.11.2. The hell?

**Comment by exaskye** [comment]
Fix published as [0.1.3](https://github.com/elytra/FruitPhone/releases/tag/1.10.2-0.1.3).

**Comment by Tylian** [comment]
> It's not client-only in 1.11.2. The hell?

Yeah I'm honestly not sure why it is either, but I just doubled checked and it is indeed client only.
That's dumb.

Ah well, thanks for the quick fix!
